components/automate-cli/pkg/verifyserver/models/batchcheck.go

```diff
@@ -123,6 +123,19 @@ func appendCertsByIpToNodeCerts(certsByIP *[]config.CertByIP, ipList []string, p
 
 }
 
+func NewConfig() *Config {
+	return &Config{
+		SSHUser: &SSHUser{},
+		Backup: &Backup{
+			FileSystem:    &FileSystem{},
+			ObjectStorage: &ObjectStorage{},
+		},
+		Hardware:   &Hardware{},
```

Comment:
We should add all the objects initialisation, like certificate?

Reply:
They will take their respective empty values, we just have to have types of objects initialized...